### ISSUE TITLE
Absolute Path

### ISSUE THREAD

**Issue by bomalley** [cnjon/react-native-pdf-view#10]
Can you please show an example that refers to a PDF in your project bundle?  I'm stuck on the 'absolute path' for the `src` attribute.

if I have this layout, what should be in my src attribute?

/index.android.js
/My.pdf

Thanks.

**Comment by bomalley** [comment]
Sorry to be so persistent, but I'm really struggling here.  Can anyone help?

Thanks in advance!

**Comment by sibeliusseraphini** [comment]
You should try to get the bundle path

Take a look at this package https://github.com/johanneslumpe/react-native-fs

`RNFS.MainBundlePath` + '/My.pdf'

**Comment by bomalley** [comment]
That's very helpful, thank you @sibeliusseraphini.

Unfortunately that only seems to work if the PDF is in the `files` folder.  If it's bundled in your APK, I can't see any way to get at it.  Am I just missing something obvious?

**Comment by sibeliusseraphini** [comment]
@bomalley how do you get the path information using Object-C/Swift on iOS, and in Java on Android?

**Comment by bomalley** [comment]
@sibeliusseraphini I'm trying to use the `react-native-fs` you recommended.  It will allow me to get the data folder, but not the bundle/package folder, which is the problem.

**Comment by sibeliusseraphini** [comment]
have you tried `MainBundlePath`? What is the result of `RNFS.readDir`?

In iOS you can use this to get the bundlePath: 

```objective-c
NSString *path = [[NSBundle mainBundle] pathForResource:@"SomeFile" ofType:@"jpeg"];
```

**Comment by bomalley** [comment]
Okay, it took A LOT of tinkering, but I've finally figured out how to display a PDF that is pre-bundled in your Android APK.  I documented the process on my website:  [How to display a PDF in React Native](http://bryanomalley.com/how-to-display-a-pdf-in-react-native)

In a nutshell:

1. Copy the PDF from your APK into the files folder on the Android device
2. Use `react-native-fs` to get an absolute URL to the PDF.
3. Use `react-native-pdf-view` to show the PDF.
